docs/guide/deployment/authorization-server.md

```diff
@@ -1,65 +1,158 @@
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.grpc.authorization.authority` | string | `endpoint of oauth provider` | `""` |
+| `api.grpc.authorization.audience` | string | `audience of oauth provider` | `""` |
+| `api.grpc.authorization.ownerClaim` | string | `owner claim of oauth provider` | `"sub"` |
+| `api.grpc.authorization.http.maxIdleConns` | int | `controls the maximum number of idle (keep-alive) connections across all hosts. Zero means no limit.` | `16` |
+| `api.grpc.authorization.http.maxConnsPerHost` | int | `optionally limits the total number of connections per host, including connections in the dialing, active, and idle states. On limit violation, dials will block. Zero means no limit.` | `32` |
+| `api.grpc.authorization.http.maxIdleConnsPerHost` | int | `if non-zero, controls the maximum idle (keep-alive) connections to keep per-host. If zero, DefaultMaxIdleConnsPerHost is used.` | `16` |
+| `api.grpc.authorization.http.idleConnTimeout` | string | `the maximum amount of time an idle (keep-alive) connection will remain idle before closing itself. Zero means no limit.` | `30s` |
+| `api.grpc.authorization.http.timeout` | string | `a time limit for requests made by this Client. A Timeout of zero means no timeout.` | `10s` |
+| `api.grpc.authorization.http.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.grpc.authorization.http.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.grpc.authorization.http.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.grpc.authorization.http.tls.useSystemCAPool` | bool | `use system certification pool` | `false` |
+
+### Http Connectivity
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.http.address` | string | `listen specification <host>:<port> for http client connection.` | `"0.0.0.0:9100"` |
+| `api.http.tls.caPool` | string | `file path to the root certificates in PEM format` |  `""` |
+| `api.http.tls.keyFile` | string | `file name of private key in PEM format` | `""` |
+| `api.http.tls.certFile` | string | `file name of certificate in PEM format` | `""` |
+| `api.http.tls.clientCertificateRequired` | bool | `require client certificate` | `true` |
+
+### Authorization Client for Device OAuth Provider
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `oauthClients.device.provider` | string | `value which comes from the device during the sign-up ("apn")` | `"generic"` |
+| `oauthClients.device.clientID` | string | `client id for authentication to get access token/authorization code` | `""` |
+| `oauthClients.device.clientSecret` | string | `client secret for authentication to get access token` |  `""` |
+| `oauthClients.device.scopes` | string | `Comma separated list of required scopes` | `""` |
```

Comment:
What is your preferred convention? Here you start description with capital letter.